shared/src/main/scala/amf/core/internal/errorhandling/WarningOnlyHandler.scala

```diff
@@ -7,6 +7,7 @@ import org.yaml.model.{SyamlException, YError}
 
 case class WarningOnlyHandler(parent: AMFErrorHandler) extends RuntimeWrapperErrorHandler(parent: AMFErrorHandler) {
```

Comment:
Same as JsonErrorHandler

Reply:
This is no syaml error handler, is for amf reports.